cfgov/unprocessed/css/enhancements/base.less

```diff
@@ -19,11 +19,7 @@
 */
 
 body {
-    .webfont-regular();
-
-    .respond-to-max( @bp-sm-max, {
-        overflow-x: hidden;
-    } );
+    .webfont-regular(); // Moved to CF
```

Comment:
I was confused by the fact that you left this here, even though it had moved to CF. Then I realized that it's in a later version of CF than this project has implemented. Should the comment be more explicit about that?